### DISCUSSION TITLE
Support dates in unix timestamp format

### DISCUSSION THREAD

**Discussion by greatwitenorth** [harttle/liquidjs#520] [Q&A]
When I use a unix timestamp below as the `creation_date` I get weird unexpected values for the date.

```
{{creation_date | date: "%b %e, %Y"}}
``` 

Unix timestamps are of int type while the date filter expects a string. When a unix timestamp is converted to a string, then used in `new Date(stringUnixTimestamp)` it creates an invalid date. If would be nice if the library could detect if the incoming value is a string or int and keep their types accordingly.

**Comment by harttle** [comment]
Can you provide a template to reproduce the problem? unix timestamps are expected to be supported. Here's a demo: https://liquidjs.com/playground.html#e3sgMTY1Njc2NjM2MCB8IGRhdGU6ICclWS8lbS8lZCcgfX0=,e30=
